src/garage/envs/bullet/__init__.py

```diff
@@ -10,4 +10,37 @@
 
 from garage.envs.bullet.bullet_env import BulletEnv
 
-__all__ = ['BulletEnv']
+__all__ = ['BulletEnv', 'get_bullet_env_list', 'get_unsupported_env_list']
+
+
+def get_bullet_env_list():
```

Comment:
can you make these private?

Reply:
Do you mean move them into BulletEnv()? 

We can have something like 
`@classmethod 
BulletEnv.check_env_supporeted()`
and 
`@classmethod 
BulletEnv.get_bullet_env_list()`

note:
`get_bullet_env_list` can be removed once `pybullet` is upgraded to the next release (can call `pybullet_envs.getList()` instead).

Reply:
I think Ryan just means you should put an underscore at the start of the function name (e.g. `_get_bullet_env_list`). This indicates outside users of garage shouldn't call this function.